lambdas/build-debug-credential-data/src/main/java/uk/gov/di/ipv/core/builddebugcredentialdata/BuildDebugCredentialDataHandler.java

```diff
@@ -56,11 +71,54 @@ public APIGatewayProxyResponseEvent handleRequest(
                     clientSessionDetailsDto.getGovukSigninJourneyId());
             return ApiGatewayResponseGenerator.proxyJsonResponse(
                     HttpStatus.SC_OK,
-                    userIdentityService.getUserIssuedDebugCredentials(
-                            clientSessionDetailsDto.getUserId()));
+                    getUserIssuedDebugCredentials(clientSessionDetailsDto.getUserId()));
         } catch (HttpResponseExceptionWithErrorBody e) {
             return ApiGatewayResponseGenerator.proxyJsonResponse(
                     e.getResponseCode(), e.getErrorBody());
         }
     }
+
+    public Map<String, String> getUserIssuedDebugCredentials(String userId) {
```

Comment:
```suggestion
    public Map<String, String> getVcItemDebugCredentials(String userId) {
```

Reply:
Again, I think this is a different thing. I was looking at renaming and thought just pulling it out into the only place we used it would be a good first step. Then we can figure out if this is even a thing we need, or if there is a better way of doing it.